### PR TITLE
Chore: Added gitpod support

### PR DESCRIPTION
**Pull request by Biki-das**
Signed-off-by: Biki-das <bikid475@gmail.com>

this add supports for gitpod , issue https://github.com/keptn/keptn.github.io/issues/1061 , 
@oleg-nenashev and @StackScribe , i am kind of confused with the commands to state with , any help is appreciated

### CODE REVIEW

**Comment by Biki-das** [comment]
@oleg-nenashev ping>

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Commands to start on workspace startup. See https://www.gitpod.io/docs/config-gitpod-file
+tasks:
```
Comment (Biki-das) [inline .gitpod.yml:2]: Need some directions on the commands to add

**Comment by Biki-das** [comment]
@StackScribe could you review?

**Comment by StackScribe** [comment]
This is related to https://github.com/keptn/keptn.github.io/issues/1061

File: .gitpod.yml (hunk 1 above)
Comment (oleg-nenashev) [reply .gitpod.yml:2]: I need to test, nit 100% sure on my own

Comment (Biki-das) [reply .gitpod.yml:2]: Sure!

**Comment by DavidPHirsch** [comment]
stale

**Pull request closed without merging**